src/mixins/Themeable.ts

```diff
@@ -1,8 +1,11 @@
 import { assign } from '@dojo/core/lang';
+import { Evented } from '@dojo/core/Evented';
 import { includes, find } from '@dojo/shim/array';
 import Map from '@dojo/shim/Map';
-import { Constructor, WidgetProperties, PropertiesChangeEvent } from './../interfaces';
-import { WidgetBase, onPropertiesChanged, handleDecorator } from './../WidgetBase';
+import { Constructor, DNode, WidgetProperties, PropertiesChangeEvent } from './../interfaces';
```

Comment:
I generally do `./` because it seems to be a imprinted in my keyboard muscle memory. I don't mind standardising if it's something that our linter can detect.